build.gradle

```diff
@@ -11,6 +11,7 @@ buildscript {
 allprojects {
   repositories {
     jcenter()
+    mavenCentral()
```

Comment:
JCenter is a superset of Maven Central, therefore, if JCenter is in your repository, you most likely won't need to also include mavenCentral.